docker-entrypoint.sh

```diff
@@ -8,7 +8,7 @@ fi
 
 if [ "$1" = 'crate' -a "$(id -u)" = '0' ]; then
     chown -R crate:crate /data
-    set -- gosu crate "$@"
+    set -- gosu crate:crate crate "$@"
```

Comment:
why did this change?

Reply:
`gosu` parameters changed

Reply:
that's kinda strange, because gosu version did not change.

Reply:
True, the version didn't change but I don't know how it every could have worked, these are the parameters for `gosu`:

```
Usage: gosu user-spec command [args]
   ie: gosu tianon bash
       gosu nobody:root bash -c 'whoami && id'
       gosu 1000:1 id

gosu version: 1.9 (go1.6.2 on linux/amd64; gc)
```

So the correct call is `gosu crate crate "$@"` or `gosu crate:crate crate "$@"`.

Reply:
Ah, it passes the `crate` command from the args and only sets the crate user here. Sorry, see that now.